index.html

```diff
@@ -2,45 +2,199 @@
+            <div class="mdc-icon-button__ripple"></div>
+            play_arrow
         </button>
     </div>
 
-    <div style="margin-bottom: 1em;">
-        <select id="language" onchange="updateLanguage(this.value)">
-            <option value="eng"> ENG </option>
-            <option value="slo"> SLO </option>
-        </select>
-        <input type="text" id='inputText' autofocus placeholder="Enter word" autofocus
-            onkeyup="updateNovaBeseda(this.value)" maxlength="5">
 
-    </div>
 
 
 
-    <div style="margin-bottom: 1em;">
-        Hide the word: <input type="checkbox" id="myCheck" onclick="hide()">
-    </div>
+    <script>
 
+        // Activate JS for text and button components
+        const choice = mdc.textField.MDCTextField.attachTo(document.querySelector('.mdc-text-field'))
+        mdc.textField.MDCTextFieldCharacterCounter.attachTo(document.querySelector('.mdc-text-field-character-counter'))
+        for (const btn of document.querySelectorAll('.mdc-button')) {
+            mdc.ripple.MDCRipple.attachTo(btn)
+        }
 
+        // Dynamic elements
+        const outline = document.getElementById("outline")
+        const visibility = document.getElementById("visibility")
+        const error = document.getElementById("error")
+        const language = document.getElementById("language")
 
 
-    <button id="randomWord" onclick="pickRandomWord()">
-        Pick a random word
-    </button>
-    <script>
+
+        function toggleVisibility() {
+            if (isTextVisible()) {
+
+                choice.input.setAttribute('type', 'password')
+                visibility.innerHTML = 'visibility_off'
+            } else {
+                choice.input.setAttribute('type', 'text')
+                visibility.innerHTML = 'visibility'
+            }
+        }
+
+        function isTextVisible() {
+            return choice.input.getAttribute('type') == 'text'
+        }
+
+
+        function toggleLanguage() {
+            if (jezik === 'eng') {
+                jezik = 'slo'
+            } else {
+                jezik = 'eng'
+            }
+            language.setAttribute('src', `${jezik}.png`)
+        }
 
 
         function copyLink() {
-            navigator.clipboard.writeText(link)
+            console.log('COPY')
+            if (choice.valid == true & beseda.length == 5) {
+                console.log('copy')
+                navigator.clipboard.writeText(url)
+            }
         }
 
+        function updateNovaBeseda(novaBeseda) {
+            beseda = novaBeseda.toLowerCase()
+        }
+
+        let url = ''
+
+
+        function pickRandomWord() {
+            if (jezik == 'eng') {
+                WL = ALL_WORDS
+            }
+            else if (jezik == 'slo') {
+                WL = VSE_BESEDE
+            }
+
+            randomWord = WL[Math.floor(Math.random() * (WL.length))]
+            console.log(randomWord)
+            document.getElementById('choice').value = randomWord
```

Comment:
There is already an element `choice` defined on line 95, which is a wrapper around the exact input field with extra funtionality. You should just use it like `choice.value = randomWord`. Importantly this also solves your problem with #24 .

Reply:
done